packages/terra-application/src/application-navigation/private/drawer-menu/DrawerMenuUser.module.scss

```diff
@@ -12,6 +12,40 @@
     .avatar-inner {
       z-index: 0;
     }
+
+    .drawer-menu-action-button {
+      background-color: var(--terra-application-navigation-drawer-menu-action-button-background-color, #004c76);
+      border: var(--terra-application-navigation-drawer-menu-action-button-border, 1px solid #a7aaab);
+      border-radius: var(--terra-application-navigation-drawer-menu-action-button-border-radius, 3px);
+      color: var(--terra-application-navigation-drawer-menu-action-button-color, #dedfe0);
+      font-size: 1rem;
+      outline: none;
+      overflow-wrap: break-word; /* Modern browsers */
+      padding: 3px 0.7142857143rem;
```

Comment:
Is it intentional to use rem for the right and left padding but to use px for the top and bottom padding of the button?

Reply:
I just copied the styles from the UX styles as it is.